naughty/debian-testing/8375-ipa-install-crash

```diff
@@ -0,0 +1,5 @@
+Traceback (most recent call last):
+  File "test/verify/check-system-realms", line *, in testClientCertAuthentication
+*
+testlib.Error: timeout
+wait_js_cond(!ph_is_present("#realms-join-dialog")): Error: condition did not become true
```

Comment:
Can we get a more specific pattern? This just waives any and all failure to join the domain, no?